4_arm.ipynb

```diff
@@ -90,11 +158,33 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "![title](./imgs/4_model_and_tf.png)\n",
+    "\n",
+    "Each coordinate represents the x-axis, y-axis and z-axis in the order of red, green, and blue."
```

Comment:
"Each coordinate represents the x-axis, y-axis, and z-axis in the order of red, green, and blue."